src/main/java/com/squareup/kotlinpoet/FunSpec.kt

```diff
@@ -457,7 +468,8 @@ class FunSpec private constructor(
     internal val String.isConstructor get() = this == CONSTRUCTOR
     internal val String.isAccessor get() = this.isOneOf(GETTER, SETTER)
 
-    private val EXPRESSION_BODY_PREFIX = CodeBlock.of("return ")
+    private val RETURN_EXPRESSION_BODY_PREFIX = CodeBlock.of("return ")
+    private val THROW_EXPRESSION_BODY_PREFIX = CodeBlock.of("throw")
```

Comment:
Need trailing space. Please add a test case for a function body that calls a method whose name starts with `throw` like `throwOrPrint("hi")` to ensure we never regress this case.

Reply:
Good point, missed that. 
02ae906ac6a7deded0da407a271ee8a5cedd1205